ocaml/xapi/xapi_pvs_proxy.ml

```diff
@@ -33,7 +33,13 @@ let create ~__context ~site ~vIF ~prepopulate =
 	if Db.VIF.get_currently_attached ~__context ~self:vIF then begin
 		Pvs_proxy_control.start_proxy ~__context vIF pvs_proxy;
 		if Db.PVS_proxy.get_currently_attached ~__context ~self:pvs_proxy then
-			Xapi_xenops.vif_set_pvs_proxy ~__context ~self:vIF true
+			try Xapi_xenops.vif_set_pvs_proxy ~__context ~self:vIF true
+			with _ ->
```

Comment:
I am not sure about matching an exception, if we have defined exceptions yes we can do. In all cases we do need alerts so logging them sounds good.